### PR TITLE
Remove old copy of storage TestEnv Value with fake end point in core ci.yml

### PR DESCRIPTION
**Pull request by vhvb1989**

### CODE REVIEW

File: sdk/core/ci.yml [hunk 1]
```diff
@@ -73,7 +73,6 @@ stages:
           Value: "DefaultEndpointsProtocol=https;AccountName=notReal;AccountKey=3333333333333333333333333333333333333333333333333333333333333333333333333333333333333333;EndpointSuffix=core.windows.net"
         - Name: PREMIUM_FILE_CONNECTION_STRING
           Value: "DefaultEndpointsProtocol=https;AccountName=notReal;AccountKey=3333333333333333333333333333333333333333333333333333333333333333333333333333333333333333;EndpointSuffix=core.windows.net"
-          Value: "DefaultEndpointsProtocol=https;AccountName=fake;AccountKey=X;EndpointSuffix=core.windows.net"
```
Comment (ahsonkhan) [inline sdk/core/ci.yml:76]: Nit: PR title suggestion
Remove incorrect copy of storage TestEnv Value in core CI.yml to fix CI runs

Comment (ahsonkhan) [reply sdk/core/ci.yml:76]: Actually, is this causing any errors or failures in CI runs?

**Review by ahsonkhan — APPROVED** [review]
Just tweaked the PR title. If it isn't accurate, please feel free to update :)

Otherwise, looks good.

**Review by RickWinter — APPROVED** [review]

File: sdk/core/ci.yml (hunk 1 above)
Comment (vhvb1989) [reply sdk/core/ci.yml:76]: https://dev.azure.com/azure-sdk/internal/_build?definitionId=1615

**Pull request merged**